<launch>
    <machine name="local" address="localhost" default="true"/>
    <node name="Chisel" pkg="chisel_ros" type="ChiselNode" output="screen">

        <param name="chunk_size_x" value="8"/>
        <param name="chunk_size_y" value="8"/>
        <param name="chunk_size_z" value="8"/>
        <!-- param name="truncation_dist_m" value="10.0"/ -->
        <param name="truncation_scale" value="2.5"/>
        <!-- param name="integration_weight" value="1"/ -->
        <param name="carving_dist_m" value="0.005"/>
        <param name="voxel_resolution_m" value="0.65"/>
        <param name="far_plane_dist" value="120"/>

        <param name="use_voxel_carving" value="false"/>
        <param name="use_color" value="true"/>

        <param name="base_transform" value="/world"/>
        <param name="depth_image_transform" value="/ref_frame"/>
        <param name="color_image_transform" value="/ref_frame"/>
        <param name="fusion_mode" value="DepthImage"/>
        <!-- param name="fusion_mode" value="PointCloud"/ -->

        <remap from="/depth_image" to="/blur/depth"/>
        <remap from="/depth_camera_info" to="/blur/depth_info"/>
        <!-- remap from="/color_image" to="/camera/rgb/image_rect_color"/ -->
        <remap from="/color_image" to="/blur/image_grey"/>
        <remap from="/color_camera_info" to="/blur/camera_info"/>

        <remap from="/camera/depth_registered/points" to="/blur/pc2"/>

    </node>
</launch>
